spec/system/primary_nav_spec.rb

```diff
@@ -35,18 +35,25 @@
   end
 
   context 'with the other primary nav links' do
-    it 'takes you to all applications when you click "all open applications"' do
-      click_on('All open applications')
-
-      heading_text = page.first('.govuk-heading-xl').text
-      expect(heading_text).to eq('All open applications')
-    end
-
     it 'takes you to search when you click "Search"' do
       click_on('Search')
 
       heading_text = page.first('.govuk-heading-xl').text
       expect(heading_text).to eq('Search for an application')
     end
   end
+
+  it 'takes you to all applications when you click "All open applications"' do
+    click_on('All open applications')
+
+    heading_text = page.first('.govuk-heading-xl').text
+    expect(heading_text).to eq('All open applications')
+  end
+
+  it 'takes you to closed applications when you click "Closed applications"' do
+    click_on('Closed applications')
+
+    heading_text = page.first('.govuk-heading-xl').text
+    expect(heading_text).to eq('Closed applications')
```

Comment:
Might be nice to add a spec that checks that some the corrent cases are shown as well :)